### ISSUE TITLE
3.3 Login & APT Concerns/Issue

### ISSUE THREAD

**Issue by Nebula-0-System** [smartfrigde/armcord#698]
**Describe the bug**
upon updating to update 3.3, it went through initial setup, and the package maintainer changed for Ubuntu/Linux Mint through APT, as well as the label and source leading me to be mildly paranoid about that. I uninstalled for now and wanted to double check that this is expected behaviour for the new release, and ask that it be added to the readme for the update if so.

i'm not all that concerned as APT didn't flag the package as being unsigned or not signed by GPG, nothing popped up with that, however if that is a concern, i can double check that on my end manually if needed, however it does at very least seem to be a bug of not retaining sign ins.

**To Reproduce**
1. sudo apt update
2. prompts to explicitly accept label and channel name changes
3. press y
4. sudo apt upgrade
5. install armcord v3.3
6. open app
7. it prompts setup and sign in again

**Expected behavior**
sudo apt update
install 3.3
open armcord
nothing new just opens like it normally does

**Desktop (please complete the following information):**
 - OS: Linux Mint virginia 21.3 x86_64
 - Method of installation APT package following repo instructions
 - Version 3.3.0 Orange

**Comment by SomeAspy** [comment]
Hi, 3.3 introduced some breaking changes to the config file, so needing to be setup again is expected. 
If you followed the instructions in the repo, you should be receiving the apt from apt.armcord.dev, which we maintain and host.

**Comment by smartfrigde** [comment]
> prompts to explicitly accept label and channel name changes       

Hi, I'm the one maintaining the apt repo, I'm using [aptly](https://www.aptly.info/) with few of my automated scripts which I think broke and didn't apply the channel name correctly, sorry for the mistake. It should be corrected by the next update.

> it prompts setup and sign in again

We changed internally our package name (`ArmCord` --> `armcord`) which may cause some issues on case sensitive systems and create new config files. You might have a leftover `.config/ArmCord` folder with all your older data.

> the package maintainer changed for Ubuntu/Linux Mint through APT

I updated the email in the package manifest to avoid spam emails on my main email address, it's still me.

In conclusion nothing bad is happening it's just few mistakes/bugs that will be solved in the near future.

**Comment by Nebula-0-System** [comment]
> Hi, 3.3 introduced some breaking changes to the config file, so needing to be setup again is expected. If you followed the instructions in the repo, you should be receiving the apt from apt.armcord.dev, which we maintain and host.

ok dope, that solves my concern there

> > prompts to explicitly accept label and channel name changes
> 
> Hi, I'm the one maintaining the apt repo, I'm using [aptly](https://www.aptly.info/) with few of my automated scripts which I think broke and didn't apply the channel name correctly, sorry for the mistake. It should be corrected by the next update.
>
> I updated the email in the package manifest to avoid spam emails on my main email address, it's still me.
> 
> In conclusion nothing bad is happening it's just few mistakes/bugs that will be solved in the near future.

as well as that concern, perfect, thanks for letting me know.

y'all should 110% put that on the readme or just link this one here for others.

have a great day and thanks for maintaining armcord, it's a great client, and i've loved it since i found it. keep up the awesome work <3

**Comment by SomeAspy** [comment]
Just thought I'd mention - if all goes well we will be switching to https://github.com/ArmCord/apt-repo

**Comment by Nebula-0-System** [comment]
> Just thought I'd mention - if all goes well we will be switching to https://github.com/ArmCord/apt-repo

thanks for letting me know : )
